test-it/ViaTests/ArrayConcatBug2D.via

```diff
@@ -0,0 +1,91 @@
+// Autogenerated Vireo assembly file
+// Generated on 07/02/2018 11:27:51
+//---------------------------------------------------
```

Comment:
It would be nice to have the CAR number here, so we can track this ArrayConcatBug2D.via to the source of issue.

Reply:
Ok, I went ahead and mentioned the CAR in the test comment.  I guess it doesn't hurt anything, but isn't useful to anyone outside NI.

Reply:
I thought the issue was due to the 2D array having elements of different sizes where the last row had more elements than the previous rows or something like that. I think that would be useful to have in the description. And you can drop the CAR number.